src/bin/hexyl.rs

```diff
@@ -121,9 +137,10 @@ fn run() -> Result<(), Box<dyn std::error::Error>> {
 
     let squeeze = !matches.is_present("nosqueezing");
 
+    #[allow(clippy::or_fun_call)]
```

Comment:
Same here. I'm not a fan of these `allow(clippy::…)` lines. Either we ignore clippy warnings that are completely irrelevant, or fix them.

Reply:
Sure. I can't see any reason why there would be a regression, but I figured letting this get caught in review was better than me overthinking it. :)

Reply:
Fixed.